### PR TITLE
iGenomes Rollout on "Binac" too

### PR DESCRIPTION
**Pull request by apeltzer**
Yup, we're ready!!

### CODE REVIEW

**Comment by ewels** [comment]
Tests failing because merging to `master`. Switched to `dev`.

Not sure that we need this test if we're not doing releases and we always want the latest (stable) code on `master`... Tests should run on PRs to make sure that we're not merging things that break. What do you think about removing that part of the travis config @sven1103?

**Comment by ewels** [comment]
@apeltzer - I don't think that the tests will properly rerun on the switched branch. Can you maybe pop a new commit on the end to make Travis run again please?

**Comment by ewels** [comment]
mini adventure

File: conf/binac.config [hunk 1]
```diff
@@ -17,7 +16,8 @@ process {
 }
 
 params {
-  igenomesIgnore = true
+  igenomesIgnore = false
```
Comment (ewels) [inline conf/binac.config]: I think this is the default so can be removed in the future if you want...

Comment (apeltzer) [reply conf/binac.config]: True, can drop it as well :-)

**Pull request merged**